integration/integration_test.go

```diff
@@ -284,6 +294,15 @@ func TestCache(t *testing.T) {
 			checkContainerDiffOutput(t, diff, expected)
 		})
 	}
+	if b, err := strconv.ParseBool(os.Getenv("BENCHMARK")); err == nil && b {
```

Comment:
I haven't looked super closely, but would it be possible to move this entire block into a separate function and pass in `benchmark_cache` and `benchmark_layers` as args?